lib/v2/nogizaka46/blog.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: 'https://www.nogizaka46.com/s/n46/api/list/blog',
+    });
+
+    const list = JSON.parse(response.data.slice(4).slice(0, -2)).data;
+
+    ctx.state.data = {
+        allowEmpty: true,
+        title: '乃木坂46 公式ブログ',
+        link: 'https://www.nogizaka46.com/s/n46/diary/MEMBER',
+        item:
+            list &&
+            list.map((item) => ({
+                title: item.title,
+                link: item.link,
+                pubDate: parseDate(item.date),
+                author: item.name,
+                description: item.text,
+            })),
```

Comment:
Please add trimmed link as guid to prevent https://github.com/DIYgod/RSSHub/issues/9134

Reply:
![2022-02-21 20 39 07@2x](https://user-images.githubusercontent.com/30112685/154956928-96e03de8-8aaa-42d5-adc9-8dca2c58f1f0.png)

It seems like gid is generated automatically, is it necessary to specify this?

Reply:
`guid` will use `link` by default if not specified. 

In https://github.com/DIYgod/RSSHub/issues/9134, it seems the params `ima` or other params from the URL keeps changing and causing duplicated items. As this feed uses a similar API as https://github.com/DIYgod/RSSHub/issues/9134, it is likely that this feed will suffer from the same issue.